.github/ISSUE_TEMPLATE.md

```diff
@@ -6,14 +6,17 @@ A summary of the issue.
 
 ### Steps to Reproduce
 
-1. This is the first step
-2. This is the second step, etc.
+1. (for example) Start a new project with `wagtail start myproject`
+2. Edit models.py as follows...
+3. ...
+
+(It's essential that you provide enough information for someone else to replicate the problem you're seeing. Simply describing something that's broken on your current project is not enough!)
```

Comment:
Have moved this paragraph up as suggested, in aea45e664f6820f47151accdd0a88030fdabb2da.

Re the suggested text change, I'm keen to stick as closely as possible to the format "here's the information we want you to supply". I realise it might come across as a bit blunt, and there's a natural motivation for us to make it more friendly and explain _why_ we're asking for information in a particular format - but ultimately this is supposed to be a form for people to fill in, not a "how to write a good bug report" advice document. If it gets too wordy, there's a risk that the people who _are_ good bug reporters will get annoyed by all the boilerplate text that they have to delete - and the people who _aren't_ will end up skimming over it or being put off entirely.

For that reason, my aim here is to try and fit in as much "education" as possible through the structure and wording of the form itself... inevitably there'll be things we'd like to say that don't fit into that format, but we have the ["guidelines for contributing"](https://github.com/wagtail/wagtail/blob/master/CONTRIBUTING.md) document for that.